index.js

```diff
@@ -91,8 +107,8 @@ Agent.prototype.addRequest = function addRequest(req, _opts) {
   req.shouldKeepAlive = false;
 
   // Create the `stream.Duplex` instance
-  let timeout;
-  let timedOut = false;
+  var timeout;
```

Comment:
Please no `var`s :)

Reply:
On CI we're testing Node 4 and 5, which doesn't have a proper functional let, I believe. If we drop support of Node < 6 I would agree to stop using vars!